Essentials/src/com/earth2me/essentials/Essentials.java

```diff
@@ -435,6 +435,35 @@ public void reload() {
             }
         }
 
+        // Register command permissions and no-usage message
+        boolean useBukkitPerms = settings.useBukkitPermissions();
+        for (Plugin plugin : getServer().getPluginManager().getPlugins()) {
+            if (plugin.getDescription().getMain().startsWith("com.earth2me.essentials")) {
+                for (String commandName : plugin.getDescription().getCommands().keySet()) {
```

Comment:
No need to search; `this` is already the main plugin class. E.g. `this.getDescrption() ...` will suffice for the loop.

Reply:
This is in order to provide compatibility for the other Essentials plugins as well, otherwise each would need to be patched individually which doesn't make much sense as they don't work with this anyways.

Reply:
Ah, makes sense then, forgot about that. This brings up yet another potential issue though -- reload doesn't seem to be called in the other plugins as far as I can tell. Does this work despite the other plugins loading after Essentials?

Reply:
This method is called onEnable, not onLoad so these plugins and their commands will already be available even if they aren't enabled yet.

Reply:
Got it, that was a nuance I wasn't aware of until now. That's actually fairly convenient. Thanks for explaining!